ray-operator/controllers/ray/rayservice_controller.go

```diff
@@ -686,6 +710,7 @@ func (r *RayServiceReconciler) updateServeDeployment(ctx context.Context, raySer
 // It's return values should be interpreted as
 // (Serve app healthy?, Serve app ready?, error if any)
 func (r *RayServiceReconciler) getAndCheckServeStatus(ctx context.Context, dashboardClient utils.RayDashboardClientInterface, rayServiceServeStatus *rayv1alpha1.RayServiceStatus, serveConfigType utils.RayServeConfigType, unhealthySecondThreshold *int32) (bool, bool, error) {
+	r.Log.V(1).Info("getAndCheckServeStatus start")
```

Comment:
We can print more information here.

Reply:
Sorry this was used for debugging. Removed now.